x/nexus/keeper/grpc_query.go

```diff
@@ -278,3 +278,18 @@ func (q Querier) TransferRateLimit(c context.Context, req *types.TransferRateLim
 		},
 	}, nil
 }
+
+// Message queries the general message for a given message ID
+func (q Querier) Message(c context.Context, req *types.MessageRequest) (*types.MessageResponse, error) {
+	ctx := sdk.UnwrapSDKContext(c)
+
+	id := req.ID
+
+	if msg, found := q.keeper.GetMessage(ctx, id); found {
```

Comment:
to follow our style convention, return the error first if not found?

Reply:
can't assign as part of the if statement then, but I don't really have a preference